Cargo.toml

```diff
@@ -203,6 +203,7 @@ mimalloc-safe = "0.1.52"
 nonmax = "0.5.5"
 num-bigint = "0.4.6"
 num-traits = "0.2.19"
+oxc-schemars = "0.8.22"
```

Comment:
You can rename like this `miette = { package = "oxc-miette", version = "2.2.1", features = ["fancy-no-syscall"] }` to avoid changing the import name.